tools/keycloak/sync/realm.json

```diff
@@ -1,8 +1,8 @@
 {
   "Realm": {
     "Name": "72x8v9rw",
-    "DisplayName": "MoTI PIMS Production Environment",
-    "DisplayNameHtml": "<a>MoTI PIMS Production Environment</a>",
+    "DisplayName": "MoTI PIMS PRODUCTION Environment",
```

Comment:
Are the changes in this file related to this PR?

Reply:
NO